src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
The problem is not with Tortilla but rather with the infrastructure of `angular-meteor-docs`. When Tortilla creates a new version of the tutorial, it will update `-history` branch and will create a new tag, but `angular-meteor-docs` doesn't know how to handle such a thing (yet). For now, let's create a new branch explicitly and update `angular-meteor-docs` in such a way that we won't loose previous version of the tutorial, and after that, we can add support for automatic versions management in `angular-meteor-docs`.